04-spatial-operations.Rmd

```diff
@@ -217,8 +222,16 @@ Since mid-2017 (with the release of **sf** `0.5-3`) the base R function `aggrega
 Building on the example presented the previous section (\@ref(spatial-subsetting)), we demonstrate this by aggregating the population of countries that intersect with the buffer represented by the circular `buff` object created in the previous section.
 
 ```{r}
-buff_agg = aggregate(x = africa["pop"], by = buff, FUN = sum)
+buff_agg = aggregate(x = africa[, "pop"], by = buff, FUN = sum)
+```
+<!--
+show also tidyverse way, so what you are doing is basically a spatial join and a subsequent aggregation without a grouping variable. Didactically, it might be better to present a grouping variable.
```

Comment:
Well, I am sorry to criticize your nice example! But aggregating is just a synonym for dissolving, and this is more impressive (especially spatially) if we have a grouping variable.

Reply:
I think aggregating is different from dissolving. The former operates on attributes the latter on the geometry. They can work well together, good to clarify the distinction.

Reply:
BTW yes I'm well-up for another example. Any ideas? The thinking behind this one was that it was clear and intuitive. It does not do dissolving. Maybe we could a second aggregating example with multiple features in the 'source' object.